External/imtool3D_td/imtool3D.m

```diff
@@ -613,7 +613,12 @@
             
             % set Image
             setImage(tool, varargin{:})
-
+            
+            % disable ROI tools if no image processing toolbox
```

Comment:
I was just puzzled about how this section is disabling ROI tools.

Reply:
Other than that, all OK.

Reply:
You're right, that's just a warning for now. Just didn't have time to push this investigation.
I open an issue